department-of-veterans-affairs/va.gov-cms
# VACMS-5470: Remove disable node menu item module.

## Description
See #5470

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/120018655-7b16a700-bfb5-11eb-81ae-7195b4db170b.png)

## QA steps
- As an administrator, visit `/node/14995/edit` and visually verify the `Enable in menu` checkbox no longer appears in sidebar menu widget.
- Save node, visually verify no menu related notices or errors occur.

### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
